taxi/views.py

```diff
@@ -111,7 +111,7 @@ class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
 @login_required
 def register_driver_to_car(request, pk):
     car = Car.objects.get(id=pk)
-    driver = Driver.objects.get(id=request.user.id)
+    driver = Driver.objects.get(id=request.user)
```

Comment:
no need to create variable driver here, u can use request.user instead of getting driver from db